MainProject/MainProject/OptimusPrime.cpp

```
@@ -0,0 +1,70 @@
+#include <cmath>
+#include <iostream>
+#include <string>
+
+using std::string;
+using std::cin;
+using std::cout;
+using std::endl;
+
+// Gets input from user and check this is an int
+int getInputFromUser(int* userInputAsNum) {
```

There isn't any need to get `int*` as parameter, you can just return the value as return type. And if there is an error, throw an exception instead of returning an error code